Revert "Run Dashboard container as non-root user"

sbreker opened the pull request:
This reverts commit a9d53698493df45c1f5b0daf4050484c32d05d65.

sbreker committed:
Revert "Run Dashboard container as non-root user"

This reverts commit a9d53698493df45c1f5b0daf4050484c32d05d65.

djjuhasz commented:
@sbreker there is a nginx "unprivileged" docker image we could use instead of the default "nginx:alpine" image: https://hub.docker.com/r/nginxinc/nginx-unprivileged

Alternatively the default nginx docker image can be run as non-root with some config changes: https://hub.docker.com/_/nginx "Running nginx as a non-root user"

sbreker committed:
Set trivy to ignore issue AVD-DS-0002

Ignore AVD-DS-0002 for now. Need to revisit issue of running nginx as
root user in dashboard.

djjuhasz reviewed (approved):
LGTM :+1:
